src/components/AvailabilityGrid.tsx

```diff
@@ -115,25 +115,64 @@ const Grid = ({
             const startIndex = timeArray.indexOf(timeSlot.beginning)
             let endIndex = timeArray.indexOf(timeSlot.end)
 
-            // Handle case where end time is not found in the timeArray (endIndex being -1)
-            if (endIndex === -1) {
-              endIndex = timeArray.length // Set to the end of the timeArray
+            // Instead of just setting to true, increment a counter to keep track of how many people are available at that time
```

Comment:
can you confirm this code is what you added and not a revert or a re-add of previous code please? I feel like this function keeps going back and forth from previous changes.

Reply:
The current version should be correct. I think it got reverted in a previous merge, but now it should be correct.